src/components/home/services/services.tsx

```diff
@@ -39,11 +56,11 @@ const ServicesBox: React.FC<ServicesBoxProps> = ({
 
   return (
     <Paper elevation={0}>
-      <Grid container>
-        <Grid item xs={12} sm={3}>
+      <Grid container className={classes.gridRoot}>
```

Comment:
naming convention should be really relevant
```suggestion
      <Grid container className={classes.servicesGridRoot}>
```